character/race.py

```diff
@@ -0,0 +1,32 @@
+class Race:
```

Comment:
Looks good, once you pull in my database commits, you can hook up the data for each race that way.